cmd/bench.go

```diff
@@ -123,7 +123,7 @@ func prepare(workloadDir string) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
-	_, err = Exec.Exec(string(sqls))
+	_, err = Exec.ExecAndRollback(string(sqls))
```

Comment:
IMO `Exec` is better, in the preparation stage, we usually think it will mutate the database metadata or store.

Reply:
Resolved